docs/getstarted/tips-and-tricks.md

```diff
@@ -802,6 +804,14 @@ During a merge, go to the **Source Control** view (`kb(workbench.view.scm)`) and
 
 ```bash
 git config --global merge.tool code
+git config --global mergetool.vscode.cmd 'code --wait $MERGED'
+```
+
+### Set VS Code as default diff tool
+
+```bash
+git config --global diff.tool vscode
```

Comment:
These two sections don't mirror each other. Should it be `git config --global diff.tool code`?

Reply:
The part that differs doesn't affect the functionality, but I agree it should be the same. Fixed.